src/util/simplify_expr.cpp

```diff
@@ -1365,17 +1365,12 @@ bool simplify_exprt::simplify_update(exprt &expr)
     {
       const irep_idt &component_name=
         e.get(ID_component_name);
-
-      if(!to_struct_type(value_ptr_type).
-         has_component(component_name))
+      const struct_typet &value_ptr_struct_type=to_struct_type(value_ptr_type);
```

Comment:
clang-format (several occurrences)

Reply:
Sorry, this commit isn't really a part of this PR it's just included because the code depends on a prior PR